code/modules/mob/living/simple_animal/abnormality/zayin/wellcheers.dm

```diff
@@ -220,24 +220,32 @@
 
 //Shrimple boat stuff
 /turf/open/water/deep/saltwater/extradeep
+	static_target = TRUE
 
-/turf/open/water/deep/saltwater/extradeep/Entered(atom/movable/thing, atom/oldLoc) //Drowning code - you won't make it back to the station alive.
-	if(!target_turf || is_type_in_typecache(thing, forbidden_types) || (thing.throwing && !istype(thing, /obj/item/food/fish || /obj/item/aquarium_prop )) || (thing.movement_type & (FLOATING|FLYING))) //replace this with a varient of chasm component sometime.
-		return ..()
+/turf/open/water/deep/saltwater/extradeep/WarpSunkStuff(atom/movable/thing) //Drowning code - you won't make it back to the station alive.
+	if(!target_turf)
+		var/shrimpspot = locate(/obj/effect/landmark/shrimpship) in world.contents
+		target_turf = get_turf(shrimpspot)
```

Comment:
No contingency for a lack of boat

Reply:
The only contingency i have is to delete the item.